### PR TITLE
Get Services that are Running as a User or Service Account

### PR DESCRIPTION
**Pull request by David-Knapp**
# PR Summary

One get-service.md file.

## PR Checklist

- Do you want this post to be published on a specific date?: Y/N [Specify date]:  N
- [X] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [X] PR has a meaningful title
- [X] Includes content related an open issue
  - see [Closing issues using keywords][key].
- [X] This PR is ready to merge and is not **Work in Progress**

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

**Comment by David-Knapp** [comment]
@microsoft-github-policy-service agree

File: Posts/2024/01/get-service.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+post_title: 'Get Services that are Running as a User or Service Account'
+user_login: David Knapp
```
Comment (sdwheeler) [inline Posts/2024/01/get-service.md:3]: The user_login name must match the username in WordPress.
```suggestion
user_login: david-knapp
```

**Comment by David-Knapp** [comment]
Hi Sean,

Thank you for the email.  How are you?  Happy new year.

That particular snippet is from a larger overall Pester describe block that is used to validate desired state after updating or performing maintenance on a stack.

I thought providing the Pester code might be too much, so I chopped it down and modified it to see how the pr went.

David
________________________________

**Comment by sdwheeler** [comment]
@David-Knapp We could use some Pester content. Especially if you explain the context of why and how to use it.

Think about how you would teach someone to create their first Pester test. I would welcome a post like that.

**Comment by David-Knapp** [comment]
Ok.

Cool.

Let me see what I can do.  Feel free to kick it back with any changes, etc ….  I would love to contribute to the code base and am just feeling out how to do it best.

I agree.  The more folks that use Pester, the more folks whose jobs became that much easier … and systems are more reliable too of course.

David

On Jan 11, 2024, at 10:16 AM, Sean Wheeler ***@***.***> wrote:

﻿

@David-Knapp<https://github.com/David-Knapp> We could use some Pester content. Especially if you explain the context of why and how to use it.

Think about how you would teach someone to create their first Pester test. I would welcome a post like that.

**Pull request closed without merging**